<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>youTubeAPIClass</class>
 <widget class="QMainWindow" name="youTubeAPIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>youTubeAPI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>691</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Benis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid #9babb2;
background-color:  #898989 ;</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="sear_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>321</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid;
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Benis</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>731</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1085</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="youTubeAPI.qrc"/>
 </resources>
 <connections/>
</ui>
